settings.xml

```diff
@@ -18,6 +18,18 @@
                 <gpg.keyname>${env.GPG_KEY_NAME}</gpg.keyname>
                 <gpg.passphrase>${env.GPG_PASSPHRASE}</gpg.passphrase>
             </properties>
+            <repositories>
+                <repository>
+                    <id>ossrh-snapshots</id>
+                    <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
```

Comment:
add `<releases><enabled>false</enabled></releases>`  to both repository elements. this way, Maven does not bother asking OSSRH for releases.